Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -72,6 +72,20 @@ configuration:
   name: proxy
   required: false
   type: 8
+- additionalinfo: 'The statuses of the incidents that will be fetched. Note: an incident whose status was changed to a filtered status after its creation time will not be fetched.'
```

Comment:
As part of the comment also explain that if no status is provided then incidents will all the statuses will be fetched

Reply:
Done.